nautobot_ssot/jobs/examples.py

```diff
@@ -943,6 +947,38 @@ def data_mappings(cls):
             DataMapping("Interface (local)", reverse("dcim:interface_list"), "Interface (remote)", None),
         )
 
+    def run(  # pylint: disable=too-many-arguments, arguments-differ
+        self,
+        dryrun,
+        memory_profiling,
+        target,
+        target_url,
+        target_token,
+        *args,
+        **kwargs,
+    ):
+        """Run sync."""
+        self.dryrun = dryrun
+        self.memory_profiling = memory_profiling
+        try:
+            if target:
+                self.logger.info(f"Using external integration '{target}'")
+                self.target_url = target.remote_url
+                secrets_group = target.secrets_group
+                self.target_token = secrets_group.get_secret_value(
+                    access_type=SecretsGroupAccessTypeChoices.TYPE_HTTP,
+                    secret_type=SecretsGroupSecretTypeChoices.TYPE_TOKEN,
+                )
+            else:
+                self.target_url = target_url
+                self.target_token = target_token
+        except Exception as error:
+            # TBD: Why are these exceptions swallowed?
```

Comment:
Is there a follow-up issue for this note?

Reply:
This appears to be from @snaselj but I'm not seeing one in a quick search. I've opened #521 to track it.